docs/source/index.mdx

```diff
@@ -32,3 +43,79 @@ It's easy to adopt Apollo incrementally, meaning you can set it up alongside an
     Learn more
   </Button>
 </div>
+
+## Recommended workouts
+
+<MenuWrapper>
+
+<WorkoutMenuItem
+  icon={HeadLight}
+  keyWord="Discover"
+  otherWords="Apollo"
+>
+
+* [The Apollo GraphQL platform](./intro/platform/)
+* [Why GraphQL?](./intro/benefits/)
+* [Introduction to Apollo Server](https://www.apollographql.com/docs/apollo-server/)
+* [Introduction to Apollo Client](https://www.apollographql.com/docs/react/)
+* [Overview of Apollo Graph Manager](https://www.apollographql.com/docs/graph-manager/)
+
+</WorkoutMenuItem>
+
+<WorkoutMenuItem
+  icon={Diving}
+  keyWord="Dive in"
+  otherWords="to the basics"
+>
+
+* Get started with [Apollo Server](https://www.apollographql.com/docs/apollo-server/getting-started/) and [Apollo Client](https://www.apollographql.com/docs/react/get-started/)
+* [Resolvers](https://www.apollographql.com/docs/apollo-server/data/resolvers/) and [Data sources](https://www.apollographql.com/docs/apollo-server/data/data-sources/) (Apollo Server)
+* [Queries](https://www.apollographql.com/docs/react/data/queries/) and [Mutations](https://www.apollographql.com/docs/react/data/mutations/) (Apollo Client)
+* The [Apollo CLI](https://www.apollographql.com/docs/devtools/cli/) and [VS Code extension](./devtools/editor-plugins/)
+* [Get started with Apollo Graph Manager](https://www.apollographql.com/docs/graph-manager/getting-started/)
+
+</WorkoutMenuItem>
+
+<WorkoutMenuItem
+  icon={Weightlift}
+  keyWord="Develop"
+  otherWords="your graph"
+>
+
+* Caching in [Apollo Server](https://www.apollographql.com/docs/apollo-server/performance/caching/) and [Apollo Client](https://www.apollographql.com/docs/react/caching/cache-configuration/)
+* [Using schema directives](https://www.apollographql.com/docs/apollo-server/schema/directives/) (Apollo Server)
+* [Extending Apollo Server with plugins](https://www.apollographql.com/docs/apollo-server/integrations/plugins/)
+* Mocking new features in [Apollo Server](https://www.apollographql.com/docs/apollo-server/testing/mocking/) and [Apollo Client](https://www.apollographql.com/docs/react/development-testing/client-schema-mocking/)
+* [The schema registry](https://www.apollographql.com/docs/graph-manager/schema-registry/)
```

Comment:
Every other bullet point uses gerunds. Could we make this "Registering your schema in the [schema registry]" instead?

Reply:
Also is there a good reason for this to be the last bullet point? If we want to promote schema registration as a key part of developing your graph, I think it should be higher on the list.